src/sidebar/services/streamer.js

```diff
@@ -81,32 +65,27 @@ export default function Streamer(
   }
 
   function handleSocketOnMessage(event) {
-    // Wrap message dispatches in $rootScope.$apply() so that
-    // scope watches on app state affected by the received message
-    // are updated
-    $rootScope.$apply(function () {
```

Comment:
The change here just removes a `$rootScope.$apply(...)` "wrapper" around the code. Historically this was needed to update the UI after WebSocket messages were processed, but this is obsolete now as Redux store subscribers handle updating the UI after processing actions.